### PR TITLE
Prevent displaying image in menu when no image uploaded

### PR DESCRIPTION
**Pull request by ErnestsVerins**

### CODE REVIEW

File: src/app/component/MenuOverlay/MenuOverlay.component.js [hunk 1]
```diff
@@ -71,15 +71,19 @@ export default class MenuOverlay extends PureComponent {
     renderItemContent(item, mods = {}) {
         const { title, icon, item_class } = item;
         const itemMods = item_class === 'MenuOverlay-ItemFigure_type_banner' ? { type: 'banner' } : mods;
+        const itemModsClass = item_class ? { ...itemMods, [item_class]: true } : itemMods;
```
Comment (alfredsgenkins) [inline src/app/component/MenuOverlay/MenuOverlay.component.js]: Please move this into separate function

File: src/app/component/MenuOverlay/MenuOverlay.style.scss [hunk 2]
```diff
@@ -289,6 +289,12 @@
 
             &_banner {
                 margin-top: 1rem;
+
+                > [class$='NoImage'] {
```
Comment (alfredsgenkins) [inline src/app/component/MenuOverlay/MenuOverlay.style.scss]: I would add a modificator to no image, instead of this construction.

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/app/component/MenuOverlay/MenuOverlay.component.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4433567&bid=15001747)

File: src/app/component/MenuOverlay/MenuOverlay.style.scss [hunk 3]
```diff
@@ -142,9 +142,8 @@
                     width: 100%;
                     height: 100%;
                     padding: calc(var(--header-height) + 20px) 1rem 1rem;
-                    box-shadow:
-                        inset 0 var(--header-height) var(--overlay-background),
-                        inset 0 calc(var(--header-height) + 1px) var(--overlay-divider-color);
+                    box-shadow: inset 0 var(--header-height) var(--overlay-background),
```
Comment (Danpiel) [inline src/app/component/MenuOverlay/MenuOverlay.style.scss]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4433567)

File: src/app/component/MenuOverlay/MenuOverlay.style.scss [hunk 4]
```diff
@@ -142,9 +142,8 @@
...
+                    inset 0 calc(var(--header-height) + 1px) var(--overlay-divider-color);
```
Comment (Danpiel) [inline src/app/component/MenuOverlay/MenuOverlay.style.scss]: Codacy Issue found: [Expected indentation of 24 spaces (indentation)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4433567)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/app/component/MenuOverlay/MenuOverlay.component.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4433567&bid=15001747)

**Comment by alfredsgenkins** [comment]
@all-contributors please add @ErnestsVerins for code

**Comment by alfredsgenkins** [comment]
Implementation does not fit my requirements, this must be re-implemented.

**Pull request closed without merging**